modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json

```diff
@@ -396,9 +396,10 @@
 	"the-mac-must-not-have-spaces": "The MAC must not have spaces.",
 	"the-project's-support-level-is-displayed-here-for-projects-with-ticketing-support": "The project's Support Level is displayed here for projects with ticketing support.",
 	"the-requested-activation-key-exceeds-the-purchased-subscriptions-for-this-liferay-project-in-case-of-unauthorized-use-liferay-can-request-financial-compensation-for-breach-of-use": "The requested activation key exceeds the purchased subscriptions for this Liferay project. In case of unauthorized use, Liferay can request financial compensation for breach of use. For more information, please reach out to your Liferay contact or sales@liferay.com.",
-	"the-requested-activation-key-is-not-yet-available-for-more-information-about-the-availability-of-your-x-activation-keys-please": "The requested activation key is not yet available. For more information about the availability of your {0} activation keys, please",
-	"the-requested-activation-key-is-not-yet-available-if-you-need-more-information-about-the-availability-of-your-x-activation-keys-please-ask-one-of-your-administrator-team-members-to-update-your-permissions-so-you-can-contact-liferay-support-alternatively-team-members-with-administrator-or-requester-role-can-submit-a-support-ticket-on-your-behalf": "The requested activation key is not yet available. If you need more information about the availability of your {0} activation keys, please ask one of your Administrator team members to update your permissions, so you can contact Liferay Support. Alternatively, team members with Administrator or Requester role can submit a support ticket on your behalf.",
 	"the-start-date-must-be-less-than": "The start date must be less than 30 days ago.",
+	"the-requested-activation-key-is-not-yet-available": "The requested activation key is not yet available.",
+	"the-requested-activation-key-is-not-yet-available-for-more-information-about-the-availability-of-your-x-activation-keys-please": " For more information about the availability of your {0} activation keys, please ",
```

Comment:
Since you spliced the text into another translation, we can remove this text

```suggestion
	"for-more-information-about-the-availability-of-your-x-activation-keys-please": " For more information about the availability of your {0} activation keys, please ",
```